### PR TITLE
Add "encoding" field for filebeat.

### PR DESCRIPTION
**Pull request by bingzijian**
Add "encoding" field for filebeat.

### CODE REVIEW

File: src/web/collector-configuration/EditInputFields.jsx [hunk 1]
```diff
@@ -220,6 +234,10 @@ const EditInputFields = React.createClass({
     return !isNaN(parseInt(value));
   },
 
+  _validString(value) {
+    return value;
+  },
```
Comment (mariussturm) [inline src/web/collector-configuration/EditInputFields.jsx]: This just returns the string, but a validator should return true/false

**Comment by mariussturm** [comment]
Other than the validation method, it looks good to me. :+1:

**Comment by mariussturm** [comment]
@bingzijian any reason for closing this?

**Comment by bingzijian** [comment]
@mariussturm I created a new pull request for it.

**Comment by mariussturm** [comment]
Ah ok, you can also push more commits to your branch. They will show up here automatically. Don't worry, maybe next time :)

**Comment by bingzijian** [comment]
@mariussturm Ok,thanks.

**Pull request closed without merging**